src/client.ts

```diff
@@ -1,11 +1,10 @@
 import * as React from 'react'
-import {mdx} from '@mdx-js/react'
 
 function getMDXComponent(
   code: string,
   globals?: Record<string, unknown>,
 ): React.FunctionComponent {
-  const scope = {React, mdx, ...globals}
+  const scope = {React, ...globals}
   // eslint-disable-next-line
   const fn = new Function(...Object.keys(scope), code)
   return fn(...Object.values(scope))
```

Comment:
If you want, you *could* support top-level await by using `AsyncFunction` (which is not a global, but you can get it with `(async () => {}).constructor`):

https://github.com/wooorm/xdm/blob/74ad3f2a18db53f80fae61f9d0415b4f9a85fe18/lib/run.js#L13